style.css

```diff
@@ -0,0 +1,693 @@
+@font-face {
+  font-family: goose;
+  src: url(fonts/Cocogoose_trial.otf);
+}
+
+body {
+  padding: 0;
+  margin: 0;
+  font-family: "Lato", sans-serif;
+}
+
```

Comment:
- [ ] I can see that your website present some overflow as you can see bellow :point_down: 

![222](https://user-images.githubusercontent.com/57562869/148445379-6d06e4a2-3e44-43f2-969f-de1691bc88d3.png)

- I think that can be solved but doing : 

```
* {
  overflow: hidden;
}

html {
  overflow: scroll;
  width: 100%;
}
```